app/app.js

```diff
@@ -12,8 +12,21 @@ angular.module('app', ['angular-md5'])
     var seriesVolumes = $window.seriesVolumes;
     
 
-    const INITIAL_ZERO = 0;  // Used for all zero intialised vars
+    const INITIAL_ZERO = 0;  // Used for all vars initialised to 0
+    const INITAL_ONE = 1;    // Used for counters intiialised to 1
     const BODY_PADDING = 20;
+    const LEFT_OFFSET = 200;
+    const TOP_OFFSET = 300;
+
+    //Colour constants used in multiple functions
+    const WHEEL_SIZE = 360;     //360 degrees in colour wheel
```

Comment:
This could be more descriptive

Reply:
Changed to TOP_MARGIN AND COMPLETE_COLOR_WHEEL_DEGREES respectively.  Can you think of a better name?